client/GlobalOptions.cc

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright (c) [2019] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+#include <snapper/AppUtil.h>
+
+#include "client/GlobalOptions.h"
+#include "client/utils/text.h"
+
+using namespace std;
+
+namespace snapper
+{
+    namespace cli
+    {
+
+	namespace
+	{
+	    const option OPTIONS[15] = {
+		{ "quiet",		no_argument,		0,	'q' },
+		{ "verbose",		no_argument,		0,	'v' },
+		{ "utc",		no_argument,		0,	0 },
+		{ "iso",		no_argument,		0,	0 },
+		{ "table-style",	required_argument,	0,	't' },
+		{ "machine-readable",	required_argument,	0,	0},
+		{ "csvout",		no_argument,		0,	0},
+		{ "jsonout",		no_argument,		0,	0},
+		{ "separator",		required_argument,	0,	0},
+		{ "config",		required_argument,	0,	'c' },
+		{ "no-dbus",		no_argument,		0,	0 },
+		{ "root",		required_argument,	0,	'r' },
+		{ "version",		no_argument,		0,	0 },
+		{ "help",		no_argument,		0,	0 },
```

Comment:
```suggestion
		{ "help",		no_argument,		0,	'h' },
```

Make -h a synonym for --help
    
To avoid fits of fury whenever I see this
    
    $ snapper -h
    Unknown global option '-h'.
    Try 'snapper --help' for more information.